app/src/common/shared/org/mozilla/vrbrowser/browser/SessionStore.java

```diff
@@ -991,6 +991,16 @@ private String checkYoutubeOverride(String aUri) {
             String result = aUri;
             result += aUri.contains("?") ? "&" : "?";
             result += "disable_polymer=1";
+            if (uri.getQueryParameter("app") == null) {
```

Comment:
it'd be nice to use `Uri.Builder`'s appendQueryParameter, but I can see how this will be useful if users want to override our rewritten query-string parameters here

Reply:
Yes, I did this way to allows users overriding our values